modules/backend/locals.tf

```diff
@@ -55,10 +53,9 @@ locals {
   task_memory_buffer_mb = var.task_memory_buffer_mb
   task_memory_mb = max(
     var.task_memory_mb,
-    local.collectionspace_memory_mb + local.elasticsearch_memory_mb + local.task_memory_buffer_mb
+    local.collectionspace_memory_mb + local.task_memory_buffer_mb
   )
-  temp_efs_name = "${local.resource_prefix}-temp"
-  template_path = "${path.module}/task-definition/app.json.tpl"
+  template_path = join("", ["${path.module}/task-definition/", length(local.elasticsearch_url) > 0 ? "app-plus-es.json.tpl" : "app-without-es.json.tpl"])
```

Comment:
Practically speaking I don't think we really support plus ES in the task definition anymore. It would be without any persistence for ES data (no EFS) which is basically unusable, so I'd be inclined to remove the plus ES option entirely.

Reply:
That's not quite what's going on. The only difference is the inclusion of the ES_HOST environment variable, not an actual ES container.

Reply:
I see. I assumed this was something else. The difference is the presence of the ES_HOST envvar. I think we should require `elasticsearch_url` to have less stuff.